### PR TITLE
Add to withdrawl

### PR DESCRIPTION
**Pull request by megstang**
What does this PR do? 
What does this change? 
Where should the reviewer look?

### CODE REVIEW

File: lib/bank_account.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ def withdraw(amount)
     if @balance >= amount
       @balance -= amount
     end
+    @balance >= amount
```
Comment (rtillies) [inline lib/bank_account.rb:18]: Yes, thanks for pulling this out separately from the `if` block!

File: lib/bank_account.rb [hunk 2]
```diff
@@ -15,6 +15,7 @@ def withdraw(amount)
     if @balance >= amount
       @balance -= amount
     end
```
Comment (rtillies) [inline lib/bank_account.rb:17]: This is the original `if` block.

**Review by rtillies — APPROVED** [review]
Thanks @megstang , will merge your PR now!

File: lib/bank_account.rb (hunk 1 above)
Comment (megstang) [reply lib/bank_account.rb:18]: yes yay

**Pull request merged**